File: University/lib/Dto/Zystem/ZBUMDto.dart

class ZBUMDto
{
  String ZVCONO = "";
  String ZVBRNO = "";
  String ZVUSNO = "";
  String ZVREMA = "";
  String ZVSYST = "";
  String ZVSTAT = "";
  double ZVRCST = 0;
  double ZVCRDT = 0;
  double ZVCRTM = 0;
  String ZVCRUS = "";
  double ZVCHDT = 0;
  double ZVCHTM = 0;
  String ZVCHUS = "";

  ZBUMDto({
    this.ZVCONO = "",
    this.ZVBRNO = "",
    this.ZVUSNO = "",
    this.ZVREMA = "",
    this.ZVSYST = "",
    this.ZVSTAT = "",
    this.ZVRCST = 0,
    this.ZVCRDT = 0,
    this.ZVCRTM = 0,
    this.ZVCRUS = "",
    this.ZVCHDT = 0,
    this.ZVCHTM = 0,
    this.ZVCHUS = "",
  });

  factory ZBUMDto.fromJson(Map<String, dynamic> json) {
    return ZBUMDto(
      ZVCONO: json["ZVCONO"] ?? "",
      ZVBRNO: json["ZVBRNO"] ?? "",
      ZVUSNO: json["ZVUSNO"] ?? "",
      ZVREMA: json["ZVREMA"] ?? "",
      ZVSYST: json["ZVSYST"] ?? "",
      ZVSTAT: json["ZVSTAT"] ?? "",
      ZVRCST: json["ZVRCST"] ?? 0,
      ZVCRDT: json["ZVCRDT"] ?? 0,
      ZVCRTM: json["ZVCRTM"] ?? 0,
      ZVCRUS: json["ZVCRUS"] ?? "",
      ZVCHDT: json["ZVCHDT"] ?? 0,
      ZVCHTM: json["ZVCHTM"] ?? 0,
      ZVCHUS: json["ZVCHUS"] ?? "",
    );
  }

  Map toMap()
  {
    var map = <String, dynamic>{};
    map["ZVCONO"] = ZVCONO;
    map["ZVBRNO"] = ZVBRNO;
    map["ZVUSNO"] = ZVUSNO;
    map["ZVREMA"] = ZVREMA;
    map["ZVSYST"] = ZVSYST;
    map["ZVSTAT"] = ZVSTAT;
    map["ZVRCST"] = ZVRCST;
    map["ZVCRDT"] = ZVCRDT;
    map["ZVCRTM"] = ZVCRTM;
    map["ZVCRUS"] = ZVCRUS;
    map["ZVCHDT"] = ZVCHDT;
    map["ZVCHTM"] = ZVCHTM;
    map["ZVCHUS"] = ZVCHUS;
    return map;
  }
}

File: University.Service/University.Dto/Zystem/ZBUMDto.cs
using System;
using System.Data;
using System.Collections.Generic;
using System.Runtime.Serialization;
using University.Dto.Base;

namespace University.Dto.Zystem
{
    [DataContract]
    public class ZBUMDto
    {
        #region Additional Property

        [DataMember]
        public string ZBBRNA { get; set; }

        [DataMember]
        public string ZCCONA { get; set; }
        
        [DataMember]
        public int PageNumber { get; set; }

        [DataMember]
        public int PageSize { get; set; }

        [DataMember]
        public int TotalPage { get; set; }

        [DataMember]
        public int TotalRecord { get; set; }

        #endregion

        #region General Property

        [DataMember]
        public string ZVCONO { get; set; }

        [DataMember]
        public string ZVBRNO { get; set; }

        [DataMember]
        public string ZVUSNO { get; set; }

        [DataMember]
        public string ZVREMA { get; set; }

        [DataMember]
        public string ZVSYST { get; set; }

        [DataMember]
        public string ZVSTAT { get; set; }

        [DataMember]
        public decimal ZVRCST { get; set; }

        [DataMember]
        public decimal ZVCRDT { get; set; }

        [DataMember]
        public decimal ZVCRTM { get; set; }

        [DataMember]
        public string ZVCRUS { get; set; }

        [DataMember]
        public decimal ZVCHDT { get; set; }

        [DataMember]
        public decimal ZVCHTM { get; set; }

        [DataMember]
        public string ZVCHUS { get; set; }

        #endregion
    }

    public class MapZBUMDto : Mapper<ZBUMDto>
    {
        protected override ZBUMDto PopulateItem(IDataRecord dr)
        {
            ZBUMDto dto = new ZBUMDto();
            for (int i = 0; i < dr.FieldCount; i++)
            {
                MapProperty(dto, dr.GetName(i), dr[i]);
            }
            return dto;
        }
    }
}